### PR TITLE
bugfix/16390-fixed-multiple-line-indicator-demo

### LINKED ISSUE #16390: Custom MultipleLines indicator demo isn't working
At the very bottom of t[his page](https://www.highcharts.com/docs/stock/custom-technical-indicators) see a demo
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/custom-regression-multiple-lines/

The above demo isn't working (should display multiple indicator lines).

#### Product version
Highcharts Stock 9.2.2

Regression after https://github.com/highcharts/highcharts/pull/16287

### PR DESCRIPTION
**Pull request by raf18seb**
Fixed #16390, a regression in a documentation article and demo. Multiple lines indicator didn't work after changes related to migrating MultipleLines mixin to composition.

Internal note: the demo wasn't working because, when extending the `linearregressionzones` **prorotype***, some options were being overwritten by the default MultipleLinex mixin's options. Should be extended with only the four prototype methods.

### CODE REVIEW

File: docs/stock/custom-technical-indicators.md [hunk 1]
```diff
@@ -273,14 +275,28 @@ Highcharts.seriesType(
...
-
-    drawGraph: multipleLinesMixin.drawGraph,
-    getTranslatedLinesNames: multipleLinesMixin.getTranslatedLinesNames,
-    translate: multipleLinesMixin.translate,
-    toYData: multipleLinesMixin.toYData
+    pointValKey: 'y'
   }
 );
+
+/* eslint-disable no-underscore-dangle */
```
Comment (karolkolodziej) [inline docs/stock/custom-technical-indicators.md:282]: I guess in the `.md` files we don't need to worry about `eslint`

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by karolkolodziej — APPROVED** [review]

**Review by bre1470 — APPROVED** [review]
Looks good! 👍

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**